.travis.yml

```diff
@@ -4,31 +4,29 @@ php:
     - 7.1
     - 7.0
     - 5.6
-    - 5.5
-    - 5.4
```

Comment:
Since versions 5.4 and 5.5 are not longer supported, I suggest we update the composer file too.